packages/component-library/styles/type.scss

```diff
@@ -11,30 +11,30 @@ $ca-weight-medium: 500;
 $ca-weight-semibold: 500; // Note: in Sketch, semibold is 600. But murmur has an existing value of semibold=500 that is heavily used.
 
 // Combine these into a Sass map ($ca-default-font) once node-sass includes libsass 3.5.0.beta.3 with this bug fix: https://github.com/sass/libsass/issues/2309
-$ca-default-font-family: "Open Sans", Helvetica, Arial, sans-serif;
-$ca-default-font-base-size: 0.875rem; /* 14px */
-$ca-default-font-descender-height: 0.115;
+$ca-inter-font-family: "Inter", sans-serif;
```

Comment:
Previously this was falling back to `$ca-default-font-family `which used to be Open Sans. I don't know how intentional that was. `sans-serif` is a temporary fallback until we figure out the best practice for font fallbacks (https://github.com/cultureamp/kaizen-design-system/issues/829).